### PR TITLE
chore: updating dependabot to support gradle, docker, gha and cargo packages.

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

**Comment by rajpalc7** [comment]
Hi @wadeBarnes, this is ready for review.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+      timezone: "Canada/Pacific"
+
+  # Maintain dependencies for docker
+  - package-ecosystem: "docker"
+    directory: "/aries/agents/mediator"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
```
Comment (WadeBarnes) [inline .github/dependabot.yml]: Duplicate entry.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
@rajpalc7, Along with the duplicate entry identified below, I think you'll find most of the `cargo` configuration is unnecessary due to the project being configured to use workspaces.

File: .github/dependabot.yml (hunk 1 above)
Comment (rajpalc7) [reply .github/dependabot.yml]: Thanks @WadeBarnes , should be good now.

**Review by WadeBarnes — APPROVED** [review]

**Pull request closed without merging**